pubtools/exodus/_tasks/push.py

```diff
@@ -14,48 +14,51 @@ def add_args(self):
         super(ExodusPushTask, self).add_args()
 
         self.parser.add_argument(
-            "--source",
-            help="Local path to a file or directory for sync to the exodus CDN",
-        )
-        self.parser.add_argument(
-            "--dest",
-            help="Remote destination for sync",
+            "--push-item",
```

Comment:
`push-item` wasn't a great choice because that term already has a different meaning, both internally and to users.  For most push tasks, each individual file pushed is shown in Pub's UI as a "push item" and can be queried by "pub push-item-query".  It's not good to overload the term to also mean "src and dest directory pair".

If this needs to be provided as a named argument, a couple of alternative ideas are: "--sync", "--content".

To me it seems best of all if it wouldn't need to be provided as a named argument though, like:

```
# just list the src & dest dirs. Have to validate we have an even number though.
pubtools-exodus-push src1 dest1 src2 dest2

# or could still accept them as pairs
pubtools-exodus-push src1,dest1 src2,dest2
```

It'd have to be tested if this can be made to work with self.extra_args though.  It might require the caller to pass `--` in between the src/dest pairs and the extra args.